src/Mod/Part/App/FeatureDerivedPart.h

```diff
@@ -0,0 +1,50 @@
+/***************************************************************************
+ *   Copyright (c) 2007 Werner Mayer <wmayer[at]users.sourceforge.net>     *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+
+#ifndef PART_FEATUREDERIVEDPART_H
+#define PART_FEATUREDERIVEDPART_H
+
+
+#include "PartFeature.h"
+#include <App/PropertyLinks.h>
+
+namespace Part
+{
+
+// "derived part" is an abstract part witch derive any part such as boolean or fusions result of operations between one of many children objects
```

Comment:
s/witch/which/
also maybe put the comment on 2 lines for better readability?

Reply:
OK I will do it